src/core/storage/ionicStorage/ionicStorage.test.ts

```diff
@@ -209,8 +209,8 @@ describe("Ionic Storage Module: Basic Storage Service", () => {
       BasicRecord
     );
     expect(getMock).toBeCalledWith(existingRecord.id);
-    expect(record!.type).toEqual(BasicRecord.type);
-    expect(record!.id).toEqual(existingRecord.id);
+    expect(record?.type).toEqual(BasicRecord.type);
```

Comment:
Why this change?

Reply:
Because it is violating `https://typescript-eslint.io/rules/no-non-null-assertion/` rule.

If you are using non-null assertion operator here, it mean `record`  is definitely not empty. Therefore, I think there is no difference when using `non-null assertion operator` and `optional chaining` here.

Reply:
sounds good